TransbankPosSDK/Responses/SaleResponse.cs

```diff
@@ -28,7 +28,7 @@ public class SaleResponse : LoadKeysResponse
 
         public string Ticket { set; get; }
         public int AutorizationCode { set; get; }
-        public int Ammount { set; get; }
```

Comment:
Same here. I recommend you to test the changes against the example project, if everything is backwards compatible, the you wont need to change anything in the example